amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
   avatar: String
+  """URL of the user's thumbnail image."""
   thumbnail: String
+  """Display name of the user."""
   displayName: String
+  """User's bio information."""
   bio: String
+  """User's location information."""
   location: String
+  """URL of the user's website."""
   website: AWSURL
+  """User's email address."""
   email: AWSEmail @index(name: "byEmail", queryField: "getProfileByEmail")
+  """Metadata associated with the user's profile."""
   meta: ProfileMetadata
 }
 
+"""
+Represents the status of a Colony's native token.
+
+Colonies can have different types of native tokens in various modes. Here we define some important properties that the dApp uses to enable or disable certain features or views. This is set when a Colony is created and can be changed later.
+"""
 type NativeTokenStatus {
-  unlocked: Boolean # If it's already unlocked
-  mintable: Boolean # User has permissions to mint new tokens
-  unlockable: Boolean # Token can be unlocked
+  """Whether the native token is unlocked."""
+  unlocked: Boolean
+  """Whether the user has permissions to mint new tokens."""
+  mintable: Boolean
+  """Whether the native token can be unlocked."""
+  unlockable: Boolean
 }
 
+"""
+Represents the status of a Colony.
+
+This contains important meta information about the Colony's token and other fundamental settings
+"""
 type ColonyStatus {
+  """Status information for the Colony's native token."""
   nativeToken: NativeTokenStatus
-  recovery: Boolean # if it's in recovery mode
+  """Whether the Colony is in recovery mode."""
+  recovery: Boolean
 }
 
-# Applies to both Colonies Tokens and Events, but not all fields are revlant to all
-# It does not apply to user accounts as they can live on all networks
+"""
+Represents metadata related to a blockchain event.
+"""
 type ChainMetadata {
+  """The network the event occurred on."""
   network: Network
+  """The chain ID of the event."""
   chainId: Int
+  """The transaction hash of the event."""
   transactionHash: String
+  """The log index of the event."""
   logIndex: Int
+  """The block number of the event."""
   blockNumber: Int
 }
 
+"""
+Represents metadata for a user's profile. Mostly user specific settings.
+"""
 type ProfileMetadata {
+  """List of email permissions for the user."""
   emailPermissions: [String!]!
 }
 
+"""
+Represents a contributor within the Colony Network.
+
+A contributor is a Colony member who has reputation.
+"""
 type Contributor {
+  """User data associated with the contributor."""
   user: User
+  """Reputation percentage of the contributor (of all reputation within the Colony)."""
   reputationPercentage: String
+  """Reputation amount of the contributor (as an absolute number)."""
   reputationAmount: String
 }
 
+"""
+Represents a watcher within the Colony Network.
+
+A watcher is a Colony member who doesn't have reputation.
+"""
 type Watcher {
+  """User data associated with the watcher."""
   user: User
 }
```

Comment:
On a side note, this seems a bit unnecessary, we could just return the `User` type directly